firefox_accounts/views/fxa_multi_service_dau.view.lkml

```diff
@@ -0,0 +1,58 @@
+view: fxa_multi_service_dau {
+
+  derived_table: {
+
+    sql:
+    SELECT
+      DATE(timestamp) as submission_date,
+      user_id,
+      ARRAY_AGG(DISTINCT service IGNORE NULLS ORDER BY service) as services,
+      MAX_BY(country_code, timestamp) as country_code
+    FROM
+      `mozdata.firefox_accounts.fxa_all_events`
+    WHERE service IN ('sync', 'mdn-plus', 'guardian-vpn', 'fx-monitor', 'fx-private-relay', 'pocket-web', 'amo-web', 'thunderbird-addons', 'mozilla-iam', 'moz-social', 'mozilla-support', 'pontoon', 'mozilla-hubs-dev')
+    AND event_type like 'fxa_activity%'
```

Comment:
It probably wouldn't make any practical difference in this case, but in `LIKE` expressions if you don't want an underscore to treated as a wildcard character it needs to be escaped with backslashes (two backslashes in a normal string, or one backslash in a raw string).